scipy/integrate/_bvp.py

```diff
@@ -810,6 +810,10 @@ def solve_bvp(fun, bc, x, y, p=None, S=None, fun_jac=None, bc_jac=None,
         mesh interval ``norm(r / (1 + abs(f)) < tol``, where ``norm`` is
         estimated in a root mean squared sense (using a numerical quadrature
         formula). Default is 1e-3.
+    tol_bc : float, optional
```

Comment:
I prefer `bc_tol` name slightly better, but doesn't really matter.

Reply:
I made that change to `solve_bvp`. Note that another parameter `tol_bc` still exists in function `solve_newton`.